tensorboard/webapp/plugins/npmi/store/npmi_reducers_test.ts

```diff
@@ -0,0 +1,220 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import * as actions from '../actions';
+import {reducers} from './npmi_reducers';
+import {DataLoadState} from './npmi_types';
+import {createNpmiState} from '../testing';
+
+describe('Annotations loading', () => {
```

Comment:
put everything describe in this file under `describe('npmi_reducers', () => { ... })`.

Reply:
Done